kong/plugins/rate-limiting/schema.lua

```diff
@@ -91,6 +91,7 @@ return {
           { redis_timeout = { type = "number", default = 2000, }, },
           { redis_database = { type = "integer", default = 0 }, },
           { hide_client_headers = { type = "boolean", required = true, default = false }, },
+          { local_timezone = { type = "boolean", required = false, default = false }, },
```

Comment:
What are the downsides of not making this behavior configurable?

Reply:
If it is made a default behaviour then it will be some kind of a breaking change, which may lead to confusion

Reply:
@bungle Thoughts?